<html>
    <head>
        <title>Error handling</title>
        <script>
          try{
    const result = 10/2;
    console.log("result",result);
}catch(error){
        console.error("error",result);
    }
  

    try{
    const result = 10/0;
    console.log("Result",result);
}catch(error){
        console.error("An error occured",error);
    }finally{
        console.log("This will always executes and runs ")
    }
        </script>
    </head>
</html>